auditbeat/docs/modules/auditd.asciidoc

```diff
@@ -135,6 +135,9 @@ following example shows all configuration options with their default values.
   include_raw_message: false
   include_warnings: false
   backpressure_strategy: auto
+  reload:
+    period: 30s
+    enabled: true
```

Comment:
can you update this to correspond to the new default?

Reply:
👍